compose-e2e-test.yml

```diff
@@ -0,0 +1,37 @@
+# compose for running e2e tests in CI
+version: "2"
+
+services:
+  remark42:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        - BUILDPLATFORM=linux/amd64
+        - SKIP_BACKEND_TEST=true
+        - SKIP_FRONTEND_TEST=true
+
+    image: umputun/remark42:dev
+    container_name: "remark42"
+
+    environment:
+      - REMARK_URL=http://remark42:8080
+      - SECRET=12345
+      - DEBUG=true
+      - ADMIN_PASSWD=password
+      - AUTH_DEV=true # activate local oauth "dev"
+      - ADMIN_SHARED_ID=dev_user # set admin flag for default user on local oauth2
+      - POSITIVE_SCORE=false # restricts comment's score to be only positive
+      - EDIT_TIME=5m # edit window
```

Comment:
The default value as well should be removed.

Reply:
Same here